src/main/java/specifications/boardSpecification/CreateBoardSpec.java

```diff
@@ -0,0 +1,13 @@
+package specifications.boardSpecification;
+
+import specifications.BaseSpec;
+import io.restassured.specification.RequestSpecification;
+
+public class CreateBoardSpec extends BaseSpec {
```

Comment:
Please combine request specifications into one class that extended BaseScpecs depends on Entity: Card, List, Board, ....